apps/roboshield/src/lib/data/payload.types.ts

```diff
@@ -0,0 +1,120 @@
+interface TextNode {
```

Comment:
I believe this should be generated based on the payload config instead of hard coding it.

Reply:
I have auto generated the types with my latest commit, am I missing something else ?

Reply:
@kelvinkipruto I have resorted to using Payload generated types in all sections of the Hero , I think we should be fine !